assets/css/restaurants.css

```diff
@@ -9,23 +9,25 @@
   justify-content: space-between;
   align-items: center;
   height: 6rem;
-  border: 1px solid red;
-  border-radius: 20px;
   padding: 0.5rem;
   margin: 0.5rem 0;
 }
 
-.restaurants__restaurant-container h3,
-p {
+.restaurants__restaurant-container p,
+h3 {
   margin: 0;
 }
 
-.restaurant-list {
+.restaurants__restaurant-name {
+  width: 50%;
+}
+
+.restaurants-list {
   display: flex;
   flex-direction: column;
-  /* width: 70%; */
+  width: 50%;
 }
 
-.restaurants__voting-section {
-  /* width: 30%; */
+.restaurants__vote-section {
+  width: 30%;
 }
```

Comment:
can you rewrite styles to sass?

Reply:
yeah sure

Reply:
as agreed, I am not switching to sass due to the fact that the project is overwhelmingly based on generic phoenix styles which are written in css